dev-packages/e2e-tests/test-applications/aws-lambda-layer-cjs/start-event-proxy.mjs

```diff
@@ -2,6 +2,6 @@ import { startEventProxyServer } from '@sentry-internal/test-utils';
 
 startEventProxyServer({
   port: 3031,
-  proxyServerName: 'aws-serverless-lambda-layer',
+  proxyServerName: 'aws-serverless-lambda-layer-cjs',
```

Comment:
just a small renaming of the test application to more clearly show what it is testing